test/e2e/run_aardvark_test.go

```diff
@@ -0,0 +1,160 @@
+package integration
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	. "github.com/containers/podman/v4/test/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("Podman run networking", func() {
+	var (
+		tempdir    string
+		err        error
+		podmanTest *PodmanTestIntegration
+		//		hostname, _ = os.Hostname()
+	)
+
+	IPRegex := `(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}`
+
+	BeforeEach(func() {
+		tempdir, err = CreateTempDirInTempDir()
+		if err != nil {
+			os.Exit(1)
+		}
+		podmanTest = PodmanTestCreate(tempdir)
+		podmanTest.Setup()
+		podmanTest.SeedImages()
+		SkipIfCNI(podmanTest)
+	})
+
+	AfterEach(func() {
+		podmanTest.Cleanup()
+		f := CurrentGinkgoTestDescription()
+		processTestResult(f)
+
+	})
+
+	FIt("Aardvark Test 1: One container", func() {
+		netName := createNetworkName("Test")
+		session := podmanTest.Podman([]string{"network", "create", netName})
+		session.WaitWithDefaultTimeout()
+		defer podmanTest.removeNetwork(netName)
+		Expect(session).Should(Exit(0))
+
+		ctrID := podmanTest.Podman([]string{"run", "-dt", "--name", "aone", "--network", netName, nginx})
+		ctrID.WaitWithDefaultTimeout()
+		cid := ctrID.OutputToString()
+		ip := podmanTest.Podman([]string{"exec", cid, "dig", "+short", "google.com"})
+		ip.WaitWithDefaultTimeout()
+		Expect(ip.OutputToString()).To(MatchRegexp(IPRegex))
+		Expect(ip).Should(Exit(0))
+
+		ctrIP := podmanTest.Podman([]string{"inspect", "--format", fmt.Sprintf(`"{{.NetworkSettings.Networks.%s.IPAddress}}"`, netName), cid})
+		ctrIP.WaitWithDefaultTimeout()
+		cip := ctrIP.OutputToString()
+		Expect(cip).To(MatchRegexp(IPRegex))
+		Expect(ip).Should(Exit(0))
+
+		reverseLookup := podmanTest.Podman([]string{"exec", cid, "dig", "+short", "-x", cip})
+		reverseLookup.WaitWithDefaultTimeout()
+		Expect(reverseLookup).Should(Exit(0))
+		revListArray := reverseLookup.OutputToStringArray()
```

Comment:
@baude can you PTAL here? In manual tests, the `exec $CID dig ...` command shows me the name and id entries, but for whatever reason the Array seems to be empty here.